### PR TITLE
Feature/cv

### PR DESCRIPTION
**Pull request by danieltolmon**

### CODE REVIEW

File: staff/daniel-toledo/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Daniel Toledo</title>
+</head>
+<body>
+    <header>
+        <h1>Curriculum Vitae Daniel Toledo</h1>
+        <h3>Renewable Energy Engineer and Future Javascript Programmer</h3>
+        <p>30th April 1992</p>
+        <p>Santa Catalina 18 1r, 08014 Barcelona, Spain</p>
+        <p><b>Tel :</b> +34 692461556</p>
```
Comment (lexbarba) [inline staff/daniel-toledo/cv/index.html]: The <b> element is used to define bold (text). This is to style it.
According to the HTML 5 specification, the <b> tag should be used as a LAST resort when no other tag is more appropriate. The HTML 5 specification states that headings should be denoted with the <h1> to <h6> tags, emphasized text should be denoted with the <em> tag, important text should be denoted with the <strong> tag, and marked/highlighted text should use the <mark> tag.

File: staff/daniel-toledo/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        <a href="mailto:daniel.toledomonfort@gmail.com">daniel.toledomonfort@gmail.com </a>
+        <dt><a href="https://www.linkedin.com/in/danieltoledomonfort/">Linkedin</a></dt>
+    </header>
+    <br>
+    <section>
+        <hr>
+        <h2><u>Experience</u></h2>
```
Comment (lexbarba) [inline staff/daniel-toledo/cv/index.html]: This is to style. The <u> element was deprecated in HTML 4.01. (the <u> element was used to define underlined text).
[https://www.w3schools.com/tags/tag_u.asp](url)

File: staff/daniel-toledo/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+        <br>
+        <ul>
+            <li>
+                <p><em>2017 Lisbon-Portugal</em></p>
+                <h3><b>Design Optimization of Mooring System for the WindFloat Technology</b></h3>
```
Comment (lexbarba) [inline staff/daniel-toledo/cv/index.html]: The element is used to define bold (text). This is to style it.
According to the HTML 5 specification, the tag should be used as a LAST resort when no other tag is more appropriate. The HTML 5 specification states that headings should be denoted with the

The element is used to define bold (text). This is to style it.
According to the HTML 5 specification, the tag should be used as a LAST resort when no other tag is more appropriate. The HTML 5 specification states that headings should be denoted with the to
tags, emphasized text should be denoted with the tag, important text should be denoted with the tag, and marked/highlighted text should use the tag. https://www.w3schools.com/tags/tag_b.asp

File: staff/daniel-toledo/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+                depths”
+                </p>
+            </li>
+        </ul>
+
+    </section>
+    <br>
+    <section>
+            <hr>
+            <h2><u>Education</u></h2>
```
Comment (lexbarba) [inline staff/daniel-toledo/cv/index.html]: This is to style. The element was deprecated in HTML 4.01. (the element was used to define underlined text).
https://www.w3schools.com/tags/tag_u.asp

File: staff/daniel-toledo/cv/index.html [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+                <li>
+                    <p>8 months travelling and working around South America</p>
+                </li>
+            </ul>
+
+        </section>
+    <br>
+    <hr>
+    <footer>
+        <p><em>CV crated by Daniel Toledo only using HTML5</em></p>
```
Comment (martimalek) [inline staff/daniel-toledo/cv/index.html]: Nice <em>!

**Comment by danieltolmon** [comment]
changed! I took away <b> and <u>

**Comment by danieltolmon** [comment]
changed! I delated <b> and <u>

**Pull request merged**